pygfx/renderers/wgpu/meshshader.py

```diff
@@ -308,9 +396,18 @@ def code_vertex(self):
 
             varyings.pick_id = u32(pick_id);
             varyings.pick_idx = u32(face_index);
-            var arr_pick_coords = array<vec3<f32>, 3>(vec3<f32>(1.0, 0.0, 0.0), vec3<f32>(0.0, 1.0, 0.0), vec3<f32>(0.0, 0.0, 1.0));
-            varyings.pick_coords = vec3<f32>(arr_pick_coords[sub_index]);
-
+            if {{indexer}} == 3 {
```

Comment:
Using templating (`$$ if indexer == 3`) would make the resulting shading code simpler, and likely more efficient.